includes/class-wc-register-classic-settings.php

```diff
@@ -0,0 +1,96 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Takes settings registered for WP-Admin and hooks them up to the new
+ * WC settings API so they can be accessed via REST.
+ *
+ * @version  2.7.0
+ * @since    2.7.0
+ * @package	 WooCommerce/Classes
+ * @category Class
+ */
+class WC_Register_Classic_Settings {
```

Comment:
Should this be legacy to match all the other work we've been doing deprecating things?